codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FrameCodec.java

```diff
@@ -613,11 +613,11 @@ private void onUpgradeEvent(ChannelHandlerContext ctx, UpgradeEvent evt) {
 
     private void onHttp2StreamWritabilityChanged(ChannelHandlerContext ctx, Http2FrameStream stream,
                                          @SuppressWarnings("unused") boolean writable) {
-        ctx.fireUserEventTriggered(Http2FrameStreamEvent.writabilityChanged(stream));
+        ctx.fireUserEventTriggered(((DefaultHttp2FrameStream) stream).writabilityChanged);
```

Comment:
Could change method arg to `DefaultHttp2FrameStream` and get rid of casting? current callers already have or can equivalently obtain the concrete type.